image-layout.md

```diff
@@ -115,4 +116,42 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## Index 
+
+Index is intended to be generated from the content in an image-layout after any updates have been made.
+Allows for mapping random access to the references in the image-layout, and needing only to read from the `oci-layout` once to build this map of references.
+
+**index** *array of objects*, OPTIONAL
```

Comment:
Adding `annotations` to descriptors is not directly coupled to removing it from an index.  It just means that folks who find the ref payload via `refs/` will still have access to it (and can skip loading the potentially large index JSON).  Once you have descriptor annotations, you can certainly cache them in the index as a separate performance optimization.  But *only* supporting ref annotations via the index (and not via `refs/`) seems like a strange position.